exotic/exotic.py

```diff
@@ -1935,12 +1936,14 @@ def main():
                 # check to make sure they have a sign
                 except ValueError as err:
                     print(err.args)
-                    infoDict['lat'] = input("Enter the latitude of where you observed (deg) "
-                                            "(Don't forget the sign where North is '+' and South is '-'): ")
+                    infoDict['lat'] = input("Enter the latitude (in degrees) of where you observed. "
+                                            "Don't forget the sign where North is '+' and South is '-'! "
+                                            "(Example: +50.4): ")
 
             if fileorcommandline == 1:
-                infoDict['long'] = input("Enter the longitude of where you observed (deg) "
-                                         "(Don't forget the sign where East is '+' and West is '-'): ")
+                infoDict['long'] = input("Enter the longitude (in degrees) of where you observed. "
+                                         "(Don't forget the sign where East is '+' and West is '-')! "
+                                         "(Example: -32): ")
```

Comment:
I would add a decimal number to the `-32` so that it is, e.g., something like `-32.13`. Otherwise people might accidentally think that they need to round up to the nearest integer.